### DISCUSSION TITLE
JVM Target

### DISCUSSION THREAD

**Discussion by griffenclark123** [GitLiveApp/firebase-kotlin-sdk#506] [Q&A] [answered]
Hello, 

I see that there's support for a JVM target, but I also read in the documentation that the JVM target is a port of the Android SDK. Would you run into issues/crashes/etc if you tried to use the JVM target on a desktop, i.e. if you wanted to utilize the sdk in JUnit tests?

Thanks

**Comment by nbransby** [comment, marked as answer]
Yes its a port of android to desktop

**Reply by griffenclark123** [reply]
Continuing on that... does that indicate that offline persistence is supported for JVM?

**Reply by nbransby** [reply]
for firestore yes, rtdb no (but probably not too hard to add)
